ocs_ci/helpers/managed_services.py

```diff
@@ -456,6 +458,22 @@ def verify_faas_provider_resources():
     # Verify that the security groups are set up correctly
     assert verify_worker_nodes_security_groups()
 
+    # Check the presence of catalogsource and its state
+    catsrc = OCP(kind=constants.CATSRC, namespace=config.ENV_DATA["cluster_namespace"])
+    catsrc_info = catsrc.get().get("items")[0]
+    log.info(f"Catalogsource: {catsrc_info}")
+    assert catsrc_info["spec"]["displayName"].startswith("Managed Fusion Agent")
+    assert catsrc_info["status"]["connectionState"]["lastObservedState"] == "READY"
+
+    # Check the presence of subscription
+    subscr = OCP(
+        kind=constants.CATSRC,
+        namespace=config.ENV_DATA["cluster_namespace"],
+        selector="operators.coreos.com/managed-fusion-agent.managed-fusion",
```

Comment:
Are we going to use this selector in other places as well?
If so, I suggest to move it to constants.